chia/daemon/server.py

```diff
@@ -145,8 +145,8 @@ def master_close_cb():
             self.self_hostname,
             self.daemon_port,
             max_size=50 * 1000 * 1000,
-            ping_interval=500,
-            ping_timeout=300,
+            ping_interval=25,
```

Comment:
I don't think we can drop it this low in case the python thread gets hung up but 500 may indeed be too much

Reply:
@djjaeger82, perhaps you could explain why you gave a thumbs down here and what you think should be done?  On the technical side, not the 'just merge it' side.

Reply:
Can we window the time and test what works and report back?  Seems like this is stale since June2

Reply:
I'm testing right now using interval=100 and timeout=50 on my system now. It is purely dedicated to running the GUI, absolutely nothing else and ever since I have updated to 1.2.x I have been having this issues several times a day.

Reply:
Nope, still having an issue:

2021-08-12T15:16:03.831 daemon __main__                   : INFO     Websocket exception. Closing websocket with chia_full_node code = 1006 (connection closed abnormally [internal]), no reason Traceback (most recent call last):
  File "websockets/protocol.py", line 827, in transfer_data
  File "websockets/protocol.py", line 895, in read_message
  File "websockets/protocol.py", line 971, in read_data_frame
  File "websockets/protocol.py", line 1047, in read_frame
  File "websockets/framing.py", line 105, in read
  File "asyncio/streams.py", line 723, in readexactly
  File "asyncio/streams.py", line 517, in _wait_for_data
asyncio.exceptions.CancelledError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "chia/daemon/server.py", line 172, in safe_handle
  File "websockets/protocol.py", line 439, in __aiter__
  File "websockets/protocol.py", line 509, in recv
  File "websockets/protocol.py", line 812, in ensure_open
websockets.exceptions.ConnectionClosedError: code = 1006 (connection closed abnormally [internal]), no reason

2021-08-12T15:16:10.128 farmer farmer                     : INFO     Reconnecting to peer {'host': '127.0.0.1', 'port': 8444}
2021-08-12T15:16:21.146 wallet wallet                     : INFO     Reconnecting to peer {'host': '127.0.0.1', 'port': 8444}
2021-08-12T15:16:44.130 farmer farmer                     : INFO     Reconnecting to peer {'host': '127.0.0.1', 'port': 8444}

Reply:
This does fix does not work. I have tried multiple different interval and timeout numbers and I am still having issues.